### DISCUSSION TITLE
Error attempting to update from 1.26.0 -> 1.27.0

### DISCUSSION THREAD

**Discussion by bmf7777** [Koenkk/zigbee2mqtt#13434] [General]
i received this error message after issuing update.sh ... any ideas how to fix ?

```
error: Your local changes to the following files would be overwritten by merge:
        package-lock.json
Please commit your changes or stash them before you merge.
```

**Comment by Nardol** [comment]
Strange, this file should not be modified.
But if you want to restore it, cd to the directory where Zigbee2MQTT is installed then type `git restore package-lock.json` and if restore does not work, replace by checkout.

**Comment by bmf7777** [comment]
git restore package-lock.json worked to remove error message ... thanks,
